com.unity.visualeffectgraph/Documentation~/whats-new-12.md

```diff
@@ -0,0 +1,95 @@
+
+![img](Images/banner-urp-fixed-lit-output.png)
+
+This version of the Visual Effect Graph adds support for lit outputs in the Universal Render Pipeline (URP). You can use this to create effects that can respond to the lighting in the scene.
+
+### 2D Renderer Support (Compute Capable Devices)
+
+![img](Images/banner-2d-renderer-support.png)
+
+In this version, the Visual Effect Graph has added support for the Universal Render Pipeline’s (URP) 2D Renderer. This means that you can now render effects in a 2D project and sort them along with sprites in your scene.
+
+For more information, see [Rendering in the Universal Render Pipeline](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest?/manual/rendering-in-universalrp.html).
+
+### Bounds Helpers
+
+![img](Images/banner-bounds-helpers.png)
+
+This version of the Visual Effect Graph adds helpers for setting and working with bounds. You can now record bounds in the Target GameObject window to ensure a more accurate fit. You can also set bounds automatically to make sure they are always visible.
+
+This feature helps you create accurate bounds that match their systems so that effects don’t unexpectedly disappear when the camera moves.
+
+For more information, see [Visual effect bounds](visual-effect-bounds.md).
+
+### Graphics/Compute Buffer Support
+
+![img](Images/banner-graphics-compute-buffer-support.png)
+
+VFX version 21.2 also adds support for Graphics/Compute buffers. This makes it easier to handle and transfer large amounts of data to a Visual Effect Graph. This is particularly useful for tracking multiple GameObject positions in your graph.
+
+This feature requires C# knowledge to set and handle Graphics Buffers.
+
+### Signed Distance Field Baker
+
+![img](Images/banner-sdf-baker.png)
+
+This version includes the new Signed Distance Field (SDF) Bake Tool. To access it, navigate to **Window > Visual Effects > Utilities > SDF Bake Tool**. You can use this tool to quickly turn meshes and prefabs into SDF assets which you can use to create various effects, such as custom collisions, or to make particles conform to a particular shape.
+
+For more information, see [Signed Distance Fields in the Visual Effect Graph](sdf-in-vfx-graph.md).
+
+### HDRP Decal Output
+
+
+## Updated
+
+The following is a list of improvements Unity made to the Visual Effect Graph in version 12.0, embedded in Unity 2021.2. Each entry includes a summary of the improvement and, if relevant, a link to any documentation.
+
+### Transform Shape
+
+![img](Images/banner-transform-shape.gif)
+
+This VFX graph version adds a transform to various shape blocks and operators to make them easier to use and to give you more precise control. For example, you can use this transform to rotate a Position Circle block to make the circle shape face a new direction, or to scale the shape of a sphere collision block on one axis to collide with an ellipse.
```

Comment:
Just googled it and apparently ellipsoid is the 3D version of an ellipse. Apologies, I wrote the wrong one originally, should be "ellipsoid".

Reply:
Ah, that makes more sense! Fixed.